app/controllers/exercises_controller.rb

```diff
@@ -53,4 +53,22 @@ def show
       end
     end
   end
+
+  def set_disabled_statuses
+    @course = Course.find(params[:course_id])
+    @organization = @course.organization
+    authorize! :teach, @organization
+
+    action = params[:commit] == 'Disable selected' ? :disabled! : :enabled!
+    exercise_params = params[:course][:exercises]
+    exercise_params.reject!(&:blank?)
+
+    exercises = Exercise.where(id: exercise_params)
+    exercises.each do |ex|
+      ex.send(action)
+    end
+
+    redirect_to manage_exercises_organization_course_path(@organization, @course),
+                notice: 'Selected exercises successfully updated.'
```

Comment:
Rubocop doesn't allow any other way to indent here. "Align the parameters of a method call if they span more than one line."